uss-discovery-api/swagger.yaml

```diff
@@ -55,19 +55,16 @@ securityDefinitions:
     tokenUrl: 'https://thisserver.com/oauth/token'
     flow: application
     scopes:
-      utm.nasa.gov_read.sdsp: read all USS Discovery data
-      utm.nasa.gov_write.sdsp: create and modify USS Discovery data that you own
-security:
-  - uss_discovery_security:
-      - utm.nasa.gov_read.sdsp
+      'utm.nasa.gov_read.stategic_cm':  Subject can read strategic conflict management data.
```

Comment:
I think for uss-disc we probably want specific scopes for discovery or just use read/write.operations scope.  I think the use of these scopes (conflictmanagement) seems a little loaded.

Reply:
Would like to discuss more.  
I see overlap in Grid and DS  -- the DS data object is USSInstance which is a way to localize/partition the airspace just like Grid.

Part of discussion is, will Discovery (DS) be completely replaced by Grid (or visa-versa); and if not, what is distinct about DS?   IMO DS would need to look at lot like Grid in order to replace it.  If DS has a role in UTM to augment a Grid-type implementation, then what is that role? Eg, DS's role could be providing situational awareness  (SA).  In that case the main DS data object would be SA data which has the property that it is not guaranteed to be correct each micro second.